js/src/setup-ads/ads-stepper/index.js

```diff
@@ -92,17 +87,16 @@ const AdsStepper = ( { formProps } ) => {
 					content: (
 						<AdsCampaign
 							trackingContext="setup-ads"
-							onContinue={ handleCreateCampaignContinue }
+							onContinue={ formProps.handleSubmit }
+							isLoading={ formProps.isSubmitting }
+							submitButtonText={ __(
+								'Launch paid campaign', 
+								'google-listings-and-ads' 
+							) }
```

Comment:
`AdsCampaign` component is used in 2 flow. One setting up our first paid campaign flow (here). And another for the setting up other paid campaing.

Next is once the first campaign set up you see a button in the right hand side of dashboard- `Add paid campaign` , 
![Screenshot 2024-09-06 at 18 01 35](https://github.com/user-attachments/assets/c9a604e2-9381-4475-8827-78869bca4f78)


once you click that you see a different flow where `AdsCampaign` is the first component in the flow.

![Screenshot 2024-09-06 at 18-05-51 Dashboard ‹ Google for WooCommerce ‹ Marketing ‹ WooCommerce ‹ google-woo — WooCommerce](https://github.com/user-attachments/assets/c49a8e26-f2e1-4a36-947e-caa1778847a2)

`submitButtonText` was introduced to make change to button text in a way it doesn't effect the other flow.

@joemcgill